tests/_support/Step/Acceptance/TemplateSteps.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * @package     redCORE
+ * @subpackage  Step
+ * @copyright   Copyright (C) 2008 - 2019 redCOMPONENT.com. All rights reserved.
+ * @licence     GNU General Public License version 2 or later; see LICENSE.TXT
+ */
+namespace Step\Acceptance;
+
+use Page\TemplatePage as disableToolbars;
```

Comment:
Hi @OscarIboy. I see code run good . But I think you should be change use Page\TemplatePage as Template or you can use TemplatePage direct needn't use  'as' It will perfect

Reply:
ok, now I do it.